framework/grid/GridView.php

```diff
@@ -503,7 +503,7 @@ public function renderTableRow($model, $key, $index)
         foreach ($this->columns as $column) {
             $cells[] = $column->renderDataCell($model, $key, $index);
         }
-        if ($this->rowOptions instanceof Closure) {
+        if (is_callable($this->rowOptions)) {
```

Comment:
usually not, but the change is not needed and its better to stick with the more restrictive approach. is_callable can have a lot of side effects and should be avoided if only annonymous function is expected.